lib/vector-core/src/event/estimated_json_encoded_size_of.rs

```diff
@@ -217,202 +218,202 @@ impl EstimatedJsonEncodedSizeOf for DateTime<Utc> {
             EPOCH_RFC3339_9
         };
 
-        QUOTES_SIZE + epoch.len()
+        QUOTES_SIZE + epoch.len().into()
     }
 }
 
 impl EstimatedJsonEncodedSizeOf for u8 {
     #[rustfmt::skip]
-    fn estimated_json_encoded_size_of(&self) -> usize {
+    fn estimated_json_encoded_size_of(&self) -> JsonSize {
         let v = *self;
 
         // 0 ..= 255
-        if        v <  10 { 1
-        } else if v < 100 { 2
-        } else            { 3 }
+        if        v <  10 { 1.into()
+        } else if v < 100 { 2.into()
+        } else            { 3.into() }
```

Comment:
Would it make sense to write these with a common `into`?
```suggestion
        if        v <  10 { 1
        } else if v < 100 { 2
        } else            { 3 }
        .into()
```